src/calculateRentalCost.js

```diff
@@ -20,7 +20,45 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  let discountsList = [];
+  class Discount {
+    constructor(discountDays, discountValue) {
+      this.days = discountDays;
+      this.value = discountValue;
+    }
+  }
+  function createDiscountList(dList) {
```

Comment:
do not declare functions inside function(There are reasons for it but not in this case)